index.html

```diff
@@ -3,17 +3,68 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/index.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Karma</title>
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+  <!-- I have deliberately completed this WITHOUT any IDs or Classes for Specificity/Selector Practise -->
+  <header>
+    <a href="./index.html"><img src="./img/karma-logo.svg" alt="Karma Logo"></a>
+    <nav>
+      <ul>
+        <li><a href="./index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a href="./store.html">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+      <a href="#"><img src="./level-3/menu-hamburger.svg" alt="Hamburger Menu"></a>
+    </nav>
+  </header>
+  <main>
+  <section>
+    <h1>Introducing Karma</h1>
+    <h3>Bring Wifi with you, everywhere you go.</h3>
+    <a href="#">Learn More</a>
+  </section>
+  <section>
+    <h2>Everyone needs a little Karma.</h2>
+    <ul>
+      <li>
+        <img src="./img/icon-devices.svg" alt="">
+        <span>Internet for all devices</span>
+      </li>
+      <li>
+        <img src="./img/icon-coffee.svg" alt="">
+        <span>Boost your productivity</span>
+      </li>
+      <li>
+        <img src="./img/icon-refill.svg" alt="">
+        <span>Pay as You Go</span>
+      </li>
+    </ul>
+  </section>
+  <section>
+    <div>
```

Comment:
Adding decretive images in css is ok and sometimes necessary if you dont have access to the html but adding empty divs that dont show there intention is not great and could be accidentally removed by another developer or your future self.
As well as this the img tag has tots of power to load the appropriate image for the screen size and connection speed as well as deferring loading of off screen images